checkers/regexpPattern_checker.go

```diff
@@ -0,0 +1,68 @@
+package checkers
+
+import (
+	"go/ast"
+	"go/constant"
+	"regexp"
+	"strings"
+
+	"github.com/go-lintpack/lintpack"
+	"github.com/go-lintpack/lintpack/astwalk"
+)
+
+func init() {
+	var info lintpack.CheckerInfo
+	info.Name = "regexpPattern"
+	info.Tags = []string{"diagnostic", "experimental"}
+	info.Summary = "Detects suspicious regexp patterns"
+	info.Before = "regexp.MustCompile(`google.com|yandex.ru`)"
```

Comment:
Should we look for `http`/`https`/`ftp` prefixes?

Reply:
I think it's safe and more useful not to restrict regexps to specific prefixes.